lib/migration.js

```diff
@@ -186,7 +186,7 @@ function mixinMigration(MySQL, mysql) {
     function actualize(propName, oldSettings) {
       var newSettings = m.properties[propName];
       if (newSettings && changed(newSettings, oldSettings)) {
-        var pName = self.client.escapeId(propName);
+        var pName = self.columnEscaped(model, propName);
```

Comment:
but in this case columnName will be not escaped and we have possible issue with columnName like `from`.